internal/controller/drplacementcontrol_controller_test.go

```diff
@@ -1245,6 +1245,7 @@ func verifyNSManifestWorkBackupLabelNotExist(resourceName, namespaceString, mana
 	Expect(err).NotTo(HaveOccurred())
 
 	Expect(mw).ToNot(BeNil())
+	Expect(mw.Spec.DeleteOption).ToNot(BeNil())
```

Comment:
Rename function from `verifyNSManifestWorkBackupLabelNotExist` to `verifyNSManifestWork`, as it verifies if the NS manifest is as desired.